src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -73,14 +73,14 @@ const semanticTokens = {
     // ? Keep "error" or rename to "fail" ?
     error: {
       base: "red.500",
-      light: "red.100",
+      light: { _light: "red.100", _dark: "red.900" },
       outline: { _light: "error.base", _dark: "error.light" },
       // ! Deprecating error.neutral
       neutral: { _light: "red.100", _dark: "red.900" },
     },
     success: {
       base: "green.500",
-      light: "green.100",
+      light: { _light: "green.100", _dark: "green.900" },
```

Comment:
@nloureiro `red.900` and `green.900` were marked as deprecated in the code because we removed them from the DS. Since we are still using that color in the Quiz Hub modals, should we add it back to the DS?

Take a look at the original issue https://github.com/ethereum/ethereum-org-website/issues/10905 to have more context on the current problem.